apps/slack/lambda/lib/routes/events/controller.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
You will notice eslint changes throughout the PR, for some reason these lint errors were being thrown as part of the PR build, but these files were untouched. Added these disable lines to fix the issue for now, and to not extend work too far out of scope.